### DISCUSSION TITLE
Config Auth doesn't work after October Update

### DISCUSSION THREAD

**Discussion by dajpes** [adonisjs/core#1863] [Help]
Hello everybody, after reading the docs concerning the October Update, it told me to do the following:

![image](https://user-images.githubusercontent.com/11248816/98305091-2ff06480-1f8f-11eb-80e7-c0e8c99b6dc2.png)

I tried to do it just how the docs says, but I got an IDE error:

![image](https://user-images.githubusercontent.com/11248816/98305772-814d2380-1f90-11eb-8fae-754b18cf8a39.png)

**Comment by RomainLanz** [comment]
Hey @dajpes! 👋 

Since you are having multiple issues updating your application. I'd recommend you to watch our screen-casts that update a real-world AdonisJS application.

▶ https://www.youtube.com/watch?v=6MBNw6JOaYQ

In your case, the issue is the following. You are not updating the `model` key, you are copy-pasting the entire `provider` key.
You should only replace the `model` key.

```js
// This is already existing.
provider: {
  // This is already existing.
  driver: 'lucid',

  // This replace "model: User"
  model: () => import('App/Models/User')
}
```
